doc/specs/#1625 - Android app support

```diff
@@ -0,0 +1,113 @@
+---
+author: @upintheairsheep
+created on: 2023-05-23
+last updated: 2023-05-23
+issue id: 1625
+---
+
+# Spec Title
+
+For [#1625](https://github.com/microsoft/winget-cli/issues/1625)
+
+## Abstract
+
+This feature adds Android application support to winget via Windows Subsystem for Android. The goal is to allow users to install and maintain Android applications just like they would any other type of application on winget, and for more community-driven Android apps to be availble via Windows. As a first step, this would be available as an "experimental" feature to make it available to the community and receive feedback.
+
+## Inspiration
+
+Android is by far the most used operating system on the planet, excluding a few certain regions, where iOS dominates. Many mobile developers make apps for Android, and only have a sometimes inferior web app for desktop, if they have one at all.
+
+## Solution Design
+
+### System requirements
+
+Windows 11 or newer  
+Windows Subsystem for Android
+ADB (if it ends up being the method of installation)
+
+### Android YAML manifest
+
+The "InstallerType" field in the YAML manifest file specifies the type of the application. A new InstallerType "Android" or "APK" will be added to show that a given application is an Android app.
+
+Each PWA would have its own manifest YAML file. A sample manifest file would look like this:
+
+```
+Id: FDroid.FDroid
+Version: 1337.0
+Name: F-Droid
+Publisher: F-Droid
+AppMoniker: fdroid
+License: Test
+InstallerType: Android
+Installers:
+- Arch: neutral
+  Url: https://f-droid.org/F-Droid.apk
+  InstallerType: Android
+  MinAndroidPlat: 31
```

Comment:
`MinAndroidPlat` should be called out in the spec as a new key which needs to be added to the manifest. What validation needs to be in place on this key? Does it need to be at the installer level, or can it also be at root level? Where would it fall for APKs distributed in zip files?